README.md

```diff
@@ -24,7 +24,7 @@
   code. Defaults to read-only file system access and no network access.
 	Access between V8 (unprivileged) and Golang (privileged) is only done via
```

Comment:
This line still says Golang. Shouldn't this say Rust/C++?

Reply:
Thanks - fixed.